### ISSUE TITLE
PsySH for breakpoints

### ISSUE THREAD

**Issue by heyalbert** [theofidry/PsyshBundle#5]
Start manually sounds not really nice - is their no other option? Maybe a workaround with a script than run in the background?

What you think? :-)

**Comment by theofidry** [comment]
You mean for starting the PHP build-in server? I would like to do something about that too, specifying the whole path is annoying. But IMO, the best way would be to use Symfony's command for that and find out why it does not work with it.

**Comment by heyalbert** [comment]
exactly. The dream would be that PsySH is working the same smoothness that it worked in Laravel.

**Comment by theofidry** [comment]
Issues to track:
* bobthecow/psysh/issues/162
* bobthecow/psysh/issues/121
